### DISCUSSION TITLE
Node issue.

### DISCUSSION THREAD

**Discussion by blazinga350** [atom/atom#22493] [Q&A]
I have installed Node.js and set it in both of my paths, but this error still pops up.

'"node"' is not recognized as an internal or external command,
operable program or batch file.
[Finished in 0.048s]

**Comment by icecream17** [comment]
Maybe your cmd didn't update.
Try `refreshenv`, or closing then opening cmd again (if your cmd is not still open just try again). (cmd only checks environment at start)

If that doesn't work maybe double check you set path correctly?
(This is system path - 'C:\Program Files\nodejs\' at bottom)
![image](https://user-images.githubusercontent.com/58114641/120003306-3c73f300-bf9b-11eb-8113-6f38092fc480.png)
